ui/src/components/view/ListView.vue

```diff
@@ -86,6 +86,9 @@
             <router-link :to="{ path: $route.path + '/' + record.uuid, query: { zoneid: $route.query.zoneid } }" v-else-if="record.uuid && $route.query.zoneid">{{ $t(text.toLowerCase()) }}</router-link>
             <router-link :to="{ path: $route.path }" v-else>{{ $t(text.toLowerCase()) }}</router-link>
           </span>
+          <span v-else-if="$route?.query?.tab === 'schedules'">
```

Comment:
afik chaining doesn't work in html part
```suggestion
          <span v-else-if="$route && $route.query && $route.query.tab === 'schedules'">
```